### ISSUE TITLE
Mount PFS in user directories?

### ISSUE THREAD

**Issue by ysimonson** [pachyderm/jupyterhub-pachyderm#41]
Not sure how feasible this is, but it'd be nice if we could mount PFS contents as a subdirectory in the user directory

**Comment by ysimonson** [comment]
@jdoliner do you think it'd be feasible to use fuse mounts v2 here?

**Comment by jdoliner** [comment]
Yeah I think that'd be totally feasible, you can probably test this out with v1, and get a good idea of if it would work, everything but writes should work with that. Unless you hit the weird bug, which seemed to affect gcp.

**Comment by ysimonson** [comment]
This isn't feasible because fuse can't run inside docker (it needs a kernel extension.)

**Comment by ysimonson** [comment]
One option may be to [reuse this extension](https://github.com/IBM/jupyterlab-s3-browser), and connect to s3 gateway. I'm not sure if it's feasible without some modifications, though, as the config for where to connect to appears to be statically defined via traitlets, whereas we need to discover the s3 gateway address at runtime.
